src/store.js

```diff
@@ -26,6 +26,32 @@ class Store {
     this._modelDefinitions = {}
   }
 
+  /**
+   * Toggles whether requests are forced to run sequentially.
+   *
+   * Defaults to `false`.
+   *
+   * @example
+   *    const store = new Store(new HttpAdapter({urlPrefix: '/api/'}))
+   *
+   *    const fetch1 = store.get('customer', '20')
+   *    store.serializeRequests = true
+   *    // Will not start until fetch1 completes
+   *    const fetch2 = store.get('offer', '30')
+   *    // Will not start until fetch2 (and transitively fetch1) complete
+   *    const fetch3 = store.get('whatever', '90')
+   *    store.serializeRequests = false
+   *    // Will start immediately
```

Comment:
Really? It won't wait for the serialized requests to complete, since they were queued? If that's intended behavior, we'll have to be careful about unlocking that semaphore in usage. Should it queue requests to be executed in parallel until the serialized requests are complete?

Reply:
The intended use case is a bounded promise that contains all of the racy requests. As long as the flag is set for that duration, the danger has passed by the time you turn it off. This is trying to mitigate "surprise" requests that are happening from totally different components, background services, etc. In these kind of flows, all the requests you are intending to be making can easily be serialized by a proper chain of `then`s or `await` points.